### ISSUE TITLE
Cannot load a model for WebGPU backend

### ISSUE THREAD

**Issue by tomoyukilabs** [mil-tokyo/webdnn#768]
`WebDNN.load('./output')` would fail to load a model for WebGPU backend, when trying as mentioned in [Use with Keras model](https://mil-tokyo.github.io/webdnn/docs/tutorial/keras.html) on Safari 11.0.3 and Safari TP 48 on macOS 10.13.3. The error message was like below:

> [Warning] Model loading failed for webgpu backend. Trying next backend: Can't find variable: placeholders (webdnn.js, line 1)

If possible, could you let me know how to solve it?

**Comment by Kiikurage** [comment]
So

**Comment by Kiikurage** [comment]
Sorry but I cannot reproduce the problem. Please check that you use the latest WebDNN runtime library.

I used [webdnn.js](https://github.com/mil-tokyo/webdnn/blob/master/dist/webdnn.es5.js).

```sh
$ md5 webdnn.es5.js
MD5 (webdnn.es5.js) = 7413d9c47df9320e36b0e83002d0c5c2
```

**Comment by tomoyukilabs** [comment]
Yes, I've already used WebDNN 1.2.1. MD5 hash is exactly the same as above. Is there any necessary file other than webdnn.js (or webdnn.es5.js)?

Other detailed instructions are:
- Keras 2.1.3 and Tensorflow 1.4.0 are installed
- The graph format is produced by the following steps:
  - generate `resnet50.h5` in Keras format (HDF5) with Python 3:
  ```python
  from keras.applications import resnet50
  model = resnet50.ResNet50(include_top=True, weights='imagenet')
  model.save("resnet50.h5")
  ```
  - convert `resnet50.h5` into WebDNN graph format:
  ```bash
  python3 bin/convert_keras.py resnet50.h5 --input_shape '(1,224,224,3)' --out output
  ```
- place `webdnn.js` and `output/` generated by the previous step in the web application's directory
- include `<script src="webdnn.js"></script>` in the web application
- execute `WebDNN.load('./output')` in the web application

**Comment by milhidaka** [comment]
Thank you for reporting.
I succeeded to reproduce the problem and fixed it in v1.2.2.

**Comment by tomoyukilabs** [comment]
I've confirmed it works well. Many thanks for your kind support!
